<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: #FAF7F0;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Franklin Gothic Demi Cond&quot;;
</string>
    </property>
    <property name="text">
     <string>AthloZone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 218, 212);
font: 9pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 218, 212);
font: 9pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonControlarStock">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Controlar Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonGestionarProveedores">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Gestionar Proveedores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRegistrarCompras">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Registrar Compras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRealizarVentas">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Realizar Ventas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonGestionarClientes">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Gestionar Clientes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
